OBSDOCS-1502 Logging 6.0.2 RN - OCP 4.17 (#85215)
openshift/openshift-docs: modules/log6x-rn-6.0.2.adoc

// Module included in the following assemblies:
// log6x-release-notes
:_mod-docs-content-type: REFERENCE
[id="logging-release-notes-6-0-2_{context}"]
= Logging 6.0.2
This release includes link:https://access.redhat.com/errata/RHBA-2024:10051[RHBA-2024:10051].

[id="logging-release-notes-6-0-2-bug-fixes_{context}"]
== Bug fixes

* Before this update, Loki did not correctly load some configurations, which caused issues when using Alibaba Cloud or IBM Cloud object storage. This update fixes the configuration-loading code in Loki, resolving the issue. (link:https://issues.redhat.com/browse/LOG-5325[LOG-5325])

* Before this update, the collector would discard audit log messages that exceeded the configured threshold. This modifies the audit configuration thresholds for the maximum line size as well as the number of bytes read during a read cycle. (link:https://issues.redhat.com/browse/LOG-5998[LOG-5998])

> * Before this update, the operator did not watch and reconcile resources associated with an instance of a ClusterLogForwarder like it did in prior releases. This update modifies the operator to watch and reconcile all resources it owns and creates. (link:https://issues.redhat.com/browse/LOG-6264[LOG-6264])

Reviewer: which Operator? Either way, even if you don't specify. Also I think that `ClusterLogForwarder` is a CR, right? 

```suggestion
* Before this update, the Operator did not watch and reconcile resources associated with an instance of a `ClusterLogForwarder` CR like it did in prior releases. This update modifies the Operator to watch and reconcile all resources it owns and creates. (link:https://issues.redhat.com/browse/LOG-6264[LOG-6264])
```

> * Before this update, the Cluster Logging Operator did not watch and reconcile resources associated with an instance of a ClusterLogForwarder like it did in prior releases. This update modifies the operator to watch and reconcile all resources it owns and creates. (link:https://issues.redhat.com/browse/LOG-6264[LOG-6264])

Reviewer 2: ```suggestion
* Before this update, the Cluster Logging Operator did not watch and reconcile resources associated with an instance of a ClusterLogForwarder like it did in prior releases. This update modifies the Operator to watch and reconcile all resources it owns and creates. (link:https://issues.redhat.com/browse/LOG-6264[LOG-6264])
```

Reviewer 3: Work to fix this is covered in [OBSDOCS-1534](https://issues.redhat.com/browse/OBSDOCS-1534).

> * Before this update, log events with an unknown severity level sent to Google Cloud Logging would trigger a warning in the vector collector, which would then default the severity to 'DEFAULT'. With this update, log severity levels are now standardized to match Google Cloud Logging specifications, and audit logs are assigned a severity of 'INFO'. (link:https://issues.redhat.com/browse/LOG-6296[LOG-6296])

Reviewer 2: Since `'` isn't a markup we use:
```suggestion
* Before this update, log events with an unknown severity level sent to Google Cloud Logging would trigger a warning in the vector collector, which would then default the severity to `DEFAULT`. With this update, log severity levels are now standardized to match Google Cloud Logging specifications, and audit logs are assigned a severity of `INFO`. (link:https://issues.redhat.com/browse/LOG-6296[LOG-6296])
```

Reviewer 3: Work to fix this is covered in [OBSDOCS-1534](https://issues.redhat.com/browse/OBSDOCS-1534).

* Before this update, when infrastructure namespaces were included in application inputs, the `log_type` was set as `application`. With this update, the `log_type` of infrastructure namespaces included in application inputs is set to `infrastructure`. (link:https://issues.redhat.com/browse/LOG-6354[LOG-6354])

> * Before this update, specifying syslog.enrichment added `namespace_name`, `container_name`, and `pod_name` to the messages of non-container logs. With this update, only container logs will include `namespace_name`, `container_name`, and `pod_name` in their messages when syslog.enrichment is set. (link:https://issues.redhat.com/browse/LOG-6402[LOG-6402])

Reviewer: should `syslog.enrichment` be in backticks? 

```suggestion
* Before this update, specifying syslog.enrichment added `namespace_name`, `container_name`, and `pod_name` to the messages of non-container logs. With this update, only container logs include `namespace_name`, `container_name`, and `pod_name` in their messages when syslog.enrichment is set. (link:https://issues.redhat.com/browse/LOG-6402[LOG-6402])
```

> * Before this update, the Cluster Logging Operator did not watch and reconcile resources associated with an instance of a ClusterLogForwarder like it did in prior releases. This update modifies the operator to watch and reconcile all resources it owns and creates. (link:https://issues.redhat.com/browse/LOG-6264[LOG-6264])
> 
> * Before this update, log events with an unknown severity level sent to Google Cloud Logging would trigger a warning in the vector collector, which would then default the severity to 'DEFAULT'. With this update, log severity levels are now standardized to match Google Cloud Logging specifications, and audit logs are assigned a severity of 'INFO'. (link:https://issues.redhat.com/browse/LOG-6296[LOG-6296])
> 
> * Before this update, when infrastructure namespaces were included in application inputs, the `log_type` was set as `application`. With this update, the `log_type` of infrastructure namespaces included in application inputs is set to `infrastructure`. (link:https://issues.redhat.com/browse/LOG-6354[LOG-6354])
> 
> * Before this update, specifying a value for the `syslog.enrichment` field of the ClusterLogForwarder added `namespace_name`, `container_name`, and `pod_name` to the messages of non-container logs. With this update, only container logs include `namespace_name`, `container_name`, and `pod_name` in their messages when `syslog.enrichment` is set. (link:https://issues.redhat.com/browse/LOG-6402[LOG-6402])

Reviewer 2: Looks like `ClusterLogForwarder` didn't get marked up as a CR yet (per peer-review):
```suggestion
* Before this update, the Cluster Logging Operator did not watch and reconcile resources associated with an instance of a `ClusterLogForwarder` custom resource (CR) like it did in prior releases. This update modifies the operator to watch and reconcile all resources it owns and creates. (link:https://issues.redhat.com/browse/LOG-6264[LOG-6264])

* Before this update, log events with an unknown severity level sent to Google Cloud Logging would trigger a warning in the vector collector, which would then default the severity to 'DEFAULT'. With this update, log severity levels are now standardized to match Google Cloud Logging specifications, and audit logs are assigned a severity of 'INFO'. (link:https://issues.redhat.com/browse/LOG-6296[LOG-6296])

* Before this update, when infrastructure namespaces were included in application inputs, the `log_type` was set as `application`. With this update, the `log_type` of infrastructure namespaces included in application inputs is set to `infrastructure`. (link:https://issues.redhat.com/browse/LOG-6354[LOG-6354])

* Before this update, specifying a value for the `syslog.enrichment` field of the `ClusterLogForwarder` CR added `namespace_name`, `container_name`, and `pod_name` to the messages of non-container logs. With this update, only container logs include `namespace_name`, `container_name`, and `pod_name` in their messages when `syslog.enrichment` is set. (link:https://issues.redhat.com/browse/LOG-6402[LOG-6402])
```

Reviewer 3: Work to fix this is covered in [OBSDOCS-1534](https://issues.redhat.com/browse/OBSDOCS-1534).

> [id="logging-release-notes-6-0-2-CVEs_{context}"]

Reviewer 2: All lowercase for IDs:
```suggestion
[id="logging-release-notes-6-0-2-cves_{context}"]
```

Reviewer 3: Work to fix this is covered in [OBSDOCS-1534](https://issues.redhat.com/browse/OBSDOCS-1534).
